saofile.js

```diff
@@ -115,6 +115,12 @@ module.exports = {
         { name: 'Single Page App', value: 'spa' }
       ],
       default: 'universal'
+    },
+    {
+      name: 'jsconfig',
```

Comment:
Maybe we can add a `checkbox` as `DevTools` which only includes .jsConfig for now and we can add more goodies in future.

Reply:
Good idea ! I updated my PR accordingly :slightly_smiling_face:

Reply:
Good idea! I updated my PR accordingly :slightly_smiling_face: